### DISCUSSION TITLE
Fine tuning arabert for hate speech detection

### DISCUSSION THREAD

**Discussion by 7ananAhmed** [aub-mind/arabert#176] [Q&A]
hello

I want to ask if I am using "Text-classification/Sentiment Analysis with Hyper Parameter tuning via Cross-Validation" example for hate speech detection. Is it necessary to fine-tune arabert? and how can I do it?

also, I want to ask about imbalanced data how can I handle it when using arabert model?

**Comment by WissamAntoun** [comment]
Yes, it is necessary to fine-tune arabert, since we only provide the pre-trained model.

Check the papers at OSCAT5 for ways to deal with imbalanced data https://aclanthology.org/2022.osact-1.20/, as a start regular fine-tuning works good enough
